Library/Homebrew/cmd/update-report.rb

```diff
@@ -127,15 +128,14 @@ def output_update_report
       updated = true
 
       old_tag = Settings.read "latesttag"
-
       new_tag = Utils.popen_read(
         "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
       ).lines.first.chomp
 
-      if old_tag.blank? || (new_tag == old_tag)
```

Comment:
`old_tag.blank?` feels like it should still be here in case `git tag list` fails.

Reply:
Sorry, in case the settings read fails.